packages/adp-tooling/src/base/abap/manifest-service.ts

```diff
@@ -2,7 +2,13 @@ import type { Manifest, ManifestNamespace } from '@sap-ux/project-access';
 import type { ToolsLogger } from '@sap-ux/logger';
 import type { AdpPreviewConfig, DescriptorVariant } from '../../types';
 import ZipFile from 'adm-zip';
-import { isAxiosError, type AbapServiceProvider, type Ui5AppInfoContent } from '@sap-ux/axios-extension';
+import {
+    isAxiosError,
+    type AbapServiceProvider,
+    type Ui5AppInfoContent,
+    type AxiosRequestConfig,
+    type ProviderConfiguration
+} from '@sap-ux/axios-extension';
```

Comment:
It would be cleaner to have import type { ... } from '@sap-ux/axios-extension';
```suggestion
import type {
    type AbapServiceProvider,
    type Ui5AppInfoContent,
    type AxiosRequestConfig,
    type ProviderConfiguration
} from '@sap-ux/axios-extension';
import { isAxiosError } from '@sap-ux/axios-extension';
```

Reply:
I made the change, but I think this will cause an issue in sonar scan. 
Fixed in: ea1b35309b367c193c5bbd833d96cdd9613c5293